server/private_routes.go

```diff
@@ -32,7 +32,7 @@ func PrivateRoutes(app *app.App) []*route.HandledRoute {
 			SecuredWith(authentication).
 			Handle(handlers.PostAccountsImport(app)),
 
-		route.Get("/accounts/{id:[0-9]+}").
+		route.Get("/accounts/{id}").
```

Comment:
@cainlevy, one of the tests (for available account) fails because this overrides the public route "/accounts/available". Any suggestions to work around this? Can't use the older [0-9] since it rejects accept emails. I think we ideally want to put the overriden route above this route definition, but being in separate files, it might be difficult.

Reply:
Oh yes that is awkward.

I think it's worth trying to register public routes before private routes to fix this. It sounds like the test case, as written, will ensure it remains that way. If we later run into a new issue that implies a need to reverse the order again, we'll find a new kind of solution.